shared/common-adapters/search-filter.tsx

```diff
@@ -25,7 +25,7 @@ const Kb = {
 type Props = {
   icon?: IconType | null
   focusOnMount?: boolean
-  fullWidth?: boolean
+  size: 'small' | 'full-width' // only affects desktop (https://zpl.io/aMW5AG3)
```

Comment:
@songgao what do you think? Because of Box2's props.fullWidth, I expected SearchFilter's `props.fullWidth` to make it grow to fill the width. But instead it means something else. This way it's less reminiscent of Box2.